extensions/amp-twitter/1.0/base-element.js

```diff
@@ -9,7 +9,7 @@ BaseElement['Component'] = BentoTwitter;
 /** @override */
 BaseElement['props'] = {
   'title': {attr: 'title'}, // Needed for Preact component
-  'options': {attrPrefix: 'data-'}, // Needed to render componoent upon mutation
+  'options': {attrPrefix: 'data-'}, // Needed to render component upon mutation
```

Comment:
Kind of odd to make this part of `options` in `<ProxyIframeEmbed />`. Would be simpler if it's part of `{...rest}` to propagate `loading` to the leaf iframe.

Does this work? It's okay if we pass an extraneous `options.loading` for now.

```suggestion
  'loading': {attr: 'data-loading'},
  'options': {attrPrefix: 'data-'}, // Needed to render component upon mutation
```

Reply:
Yes, including `'loading': {attr: 'data-loading'}` works. Updated.